requirements/constraints.txt

```diff
@@ -16,13 +16,8 @@
 # the next major release, ensure the installed version is within boundaries.
 celery>=5.2.2,<6.0.0
 
-# Adding pin to avoid any major upgrade.
-# 3.15.1 is the latest version that works with Django 3.2
-djangorestframework<=3.14.0
-
+#Necessary for python 3.8
```

Comment:
This library dropped support for Python 3.8 but that was not done until version 9.15.x if I'm not mistaken https://github.com/openedx/openedx-events/releases. So do we need the <= 9.0.0 to ensure python 3.8 support? Also, if we're using Python 3.8 to generate the requirements files, shouldn't that be enough to install a compatible version for Python 3.8?

Reply:
I removed the constraint since it didn't cause any major problem